src/OpenTelemetry.Contrib.Shared/Api/ExceptionExtensions.cs

```diff
@@ -28,18 +28,18 @@ internal static class ExceptionExtensions
     /// </summary>
     /// <param name="exception">Exception to convert to string.</param>
     /// <returns>Exception as string with no culture.</returns>
-    public static string ToInvariantString(this Exception exception)
+    public static string ToInvariantString(this System.Exception exception)
     {
-        var originalUICulture = Thread.CurrentThread.CurrentUICulture;
+        var originalUICulture = System.Threading.Thread.CurrentThread.CurrentUICulture;
```

Comment:
OneCollector is raising exception due to unneeded usings due to enabled `ImplicitUsings`.
It is the easiest way to fix this.